isacc_messaging/api/isacc_record_creator.py

```diff
@@ -303,12 +304,23 @@ def on_twilio_message_received(self, values):
         )
 
     def score_message(self, message):
-        model_path = current_app.config.get('TORCH_MODEL_PATH')
-        if not model_path:
+        ml_service_address = current_app.config.get('ML_SERVICE_ADDRESS')
+        if not ml_service_address:
             return "routine"
 
         try:
-            score = predict_score(message, model_path)
+            url = f'{ml_service_address}/predict_score'
+            headers = {'Content-Type': 'application/json'}
+            data = {'message': message}
+
+            response = requests.post(url, headers=headers, data=json.dumps(data))
```

Comment:
requests.post with a defined `json` parameter eliminates the work you did with headers.  this should simplify to:
```
response = requests.post(url, json={"message": message})
```